source/ja/fv_convert.rst

```diff
@@ -932,4 +935,200 @@ binary_typesで指定できる。`OpenCV <https://github.com/opencv>`_ を利用
           "y_size":"120.0"
           "function": "create",
         }
-      }
\ No newline at end of file
+      }
+
+Python Bridge
+-------------
+
+Python Bridge を使用すると、C++ でプラグインを作成しなくても、特徴抽出ロジックを Python で記述することができる。
+現在、以下のブリッジインタフェースが利用できる。
+
+   ==================== ====================
+   インタフェース       用途
+   ==================== ====================
+   ``string_feature``   文字列に対する特徴抽出。 (``string_types``)
+   ``word_splitter``    文字列に対する特徴抽出 (部分文字列の抽出のみ)。 (``string_types``)
+   ``num_feature``      数値に対する特徴抽出。 (``num_types``)
+   ``binary_feature``   バリナリデータに対する特徴抽出。 (``binary_types``)
+   ==================== ====================
+
+設定
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Python Bridge はプラグイン形式で提供されている。
+
+.. describe:: libpython_bridge.so
+
+ ``string_types``, ``num_types``, ``binary_types`` で指定できる。
+ ``--enable-python-plugin`` または ``--enable-python3-plugin`` オプション付きでコンパイルした場合のみ利用可能である。
+ バイナリパッケージ (RHEL 6 を除く) でもこのプラグインは利用可能である。
+
+  :function:  上記のインタフェース名を指定する (例: ``string_feature``)。
+  :module:    Pythonモジュール名を指定する。
+  :class:     Pythonモジュール内のクラス名を指定する。
+
+ .. code-block:: js
+
+      "binary_types": {
+        "extract_length": {
+          "method": "dynamic",
+          "path": "libpython_bridge.so",
+          "function": "binary_feature",
+          "module": "binary_length",
+          "class": "BinaryLengthExtractor"
+        }
+      }
+
+上記に加えて、Pythonクラスに渡す任意のパラメタを指定することができる。
+パラメタのキーとバリューは共に文字列型である必要がある。
```

Comment:
In this example, other parameters are not exist and users may think where is the parameter.
I think `multiplier_by_3` example is appropriate here.